app/javascript/widget/composables/useConfig.js

```diff
@@ -0,0 +1,104 @@
+import { computed } from 'vue';
+
+/**
+ * Composable for handling config-related computations.
+ * @returns {Object} An object containing computed properties for various configuration settings.
+ */
+export function useConfig() {
+  /**
+   * Computed property to get the channel config.
+   * @returns {Object} The channel configuration object.
+   */
+  const channelConfig = computed(() => window.chatwootWebChannel);
+
+  /**
+   * Computed property to check if the inbox avatar is enabled for the bot.
+   * @returns {Boolean} True if the inbox avatar is enabled for the bot, false otherwise.
+   */
+  const useInboxAvatarForBot = computed(() =>
+    channelConfig.value.enabledFeatures.includes('use_inbox_avatar_for_bot')
```

Comment:
@fayazara Can we create a common method for checking if a feature is enabled, since the getter is accessing values from the same object in lot of places.

```js
const createFeatureChecker = feature => computed(() => channelConfig.value.enabledFeatures.includes(feature));
const useInboxAvatarForBot = createFeatureChecker('use_inbox_avatar_for_bot');
```

Reply:
@muhsin-k fixed this, PTAL

Reply:
Looks good. Can you fix the conflicts?